### ISSUE TITLE
Application crash when try to open raw file.

### ISSUE THREAD

**Issue by star-lapsi** [nomacs/nomacs#812]
**Describe the bug**
When I try to open a raw file(from cellphone, *.dng) in Win10, applications always open and crash down immediately. Check the system logs, it shows "Application Errow" about Kernelbase.dll (which from c:\windows\system32). But I can open these files by microsoft photo(with raw file support).
Nomacs Version: 3.16.1709(x64).

**To Reproduce**
Everytime open the raw files, including open from app or click right buttom.

**Screenshots**
System log:
![图片](https://user-images.githubusercontent.com/90501831/134901442-6c4cb148-032a-414d-9ea5-2055cc05a42b.png)
![图片](https://user-images.githubusercontent.com/90501831/134901111-c56609c3-2194-41e3-9d78-7a41f36abc11.png)

**Desktop (please complete the following information):**
 - OS: [Windows 10 Home]
 - Version [21H1]

**Additional context**
Some information about RAW file and equipment: 
Cellphone : Xiaomi K30 Pro.
Cellphone OS: MIUI 12.5.5(Android 11).
Raw file for example:
[IMG_20201024_143515.zip](https://github.com/nomacs/nomacs/files/7235939/IMG_20201024_143515.zip)

Thanks for reading.

**Comment by leejuyuu** [comment]
Opening the file no longer crashes on Windows 10, with nomacs 3.19.1 (ba263741). So, please try a newer version.

However, I think the color looks off.

![image](https://github.com/user-attachments/assets/bf53f167-a57b-432e-b4e2-0e975e64faf8)

Here's how it looks on Arch Linux

![image](https://github.com/user-attachments/assets/51e84b84-71bd-4960-8a71-e875a582e94e)

@novomesk Please take a look. I'm not familiar enough with image formats...

**Comment by leejuyuu** [comment]
I think the color issue might be related to https://github.com/nomacs/nomacs/issues/574.

**Comment by star-lapsi** [comment]
@leejuyuu 
Thanks! I tried the new beta version, it never crashed again, but my system changed, it's win11 now...so I can't garuntee how it works in win10.
Color problem remains in win11, just like what you paste.
And I find a small bug (it doesn't bother me when using app) : when the win11 system is in dark mode, and nomacs interface color settings use the "system", and the "ABOUT(?)" information losing some words:
![图片](https://github.com/user-attachments/assets/7034478d-9f8b-4b16-a1b3-aac2634c9711)

**Comment by scrubbbbs** [comment]
> However, I think the color looks off.

This is #574

**Comment by leejuyuu** [comment]
@star-lapsi Thanks for testing! I think we agree that the crash has been fixed. Let's close this, and we can move to #574 to solve the color problem. Feel free to reopen if needed.

**Comment by star-lapsi** [comment]
@leejuyuu Yeah, I retry it in another device with win10 today, it's OK now - about crash. 
Thanks for your and team's work to solve this problem. Best wishes.

**Comment by leejuyuu** [comment]
That's great! Thanks for testing.
